src/App.test.js

```diff
@@ -13,11 +15,41 @@ const messages = {
   en: messages_en,
 };
 
-it('renders welcome message', () => {
-  const { getByText } = render(
-    <IntlProvider locale="en" messages={messages['en']}>
-      <App />
-    </IntlProvider>,
-  );
-  expect(getByText('Learn React')).toBeInTheDocument();
+const response = {
+  data: {
+    user: {
+      Email: 'fcoronel@makingsense.com',
+    },
+  },
+};
+
+describe('Test App component', () => {
+  afterEach(cleanup);
+
+  beforeEach(() => {
+    axios.get = jest.fn(() => Promise.resolve(response));
+  });
+
+  it('It fetches user and test welcome message', () => {
```

Comment:
I see `it('It` a bit redundant please check official documentation https://facebook.github.io/create-react-app/docs/running-tests#writing-tests

Reply:
fixed